plutus-core/plutus-core/test/CostModelInterface/defaultCostModelParams.json

```diff
@@ -144,6 +144,9 @@
   "keccak_256-cpu-arguments-intercept": 2261318,
   "keccak_256-cpu-arguments-slope": 64571,
   "keccak_256-memory-arguments": 4,
+  "ripemd_160-cpu-arguments-intercept": 1927926,
+  "ripemd_160-cpu-arguments-slope": 82523,
+  "ripemd_160-memory-arguments": 4,
```

Comment:
These numbers do not match the builtinCostModelB&C.json but only one of the 2 duplicates in A.json

Reply:
Grrr.  I thought I'd been very careful about that.

Reply:
That file's only for testing, so I think the actual numbers don't matter.  I'll change it anyway, just to be on the safe side.